core/compose/src/main/java/com/chesire/nekome/core/compose/theme/Theme.kt

```diff
@@ -10,19 +10,28 @@ import androidx.compose.runtime.Composable
 import androidx.compose.ui.platform.LocalContext
 import com.google.accompanist.systemuicontroller.rememberSystemUiController
 
+/**
+ * Used to set the theme for a given composable, defaulting to the system theme
```

Comment:
This is correct, but the ideal is that the theme is set for the view hierarchy once at the start, not for a single composable

Reply:
Addressed in commit deaa97d, now a bit less... or more? vague.